js/kucoinfutures.js

```diff
@@ -1594,4 +1596,54 @@ module.exports = class kucoinfutures extends kucoin {
     async fetchLedger (code = undefined, since = undefined, limit = undefined, params = {}) {
         throw new BadRequest (this.id + ' has no method fetchLedger');
     }
+
+    async fetchLeverageTiers (symbol = undefined, params = {}) {
+        await this.loadMarkets ();
+        if (!symbol) {
+            throw new ArgumentsRequired (this.id + '.fetchLeverageTiers requires argument symbol');
+        }
+        const market = this.market (symbol);
+        if (market['spot']) {
+            throw new BadRequest (this.id + '.fetchLeverageTiers symbol supports contract markets only');
+        }
+        const request = {
+            'symbol': market['id'],
+        };
+        const response = await this.futuresPublicGetContractsRiskLimitSymbol (this.extend (request, params));
+        //
+        //    {
+        //        "code": "200000",
+        //        "data": [
+        //            {
+        //                "symbol": "ETHUSDTM",
+        //                "level": 1,
+        //                "maxRiskLimit": 300000,
+        //                "minRiskLimit": 0,
+        //                "maxLeverage": 100,
+        //                "initialMargin": 0.0100000000,
+        //                "maintainMargin": 0.0050000000
+        //            },
+        //            ...
+        //        ]
+        //    }
+        //
+        const data = this.safeValue (response, 'data');
+        const tiers = {};
+        for (let i = 0; i < data.length; i++) {
+            const tier = data[i];
+            const symbol = this.safeSymbol (this.safeString (tier, 'symbol'));
+            if (!(symbol in tiers)) {
+                tiers[symbol] = [];
+            }
+            tiers[symbol].push ({
+                'tier': this.safeNumber (tier, 'level'),
+                'notionalFloor': this.safeNumber (tier, 'minRiskLimit'),
+                'notionalCap': this.safeNumber (tier, 'maxRiskLimit'),
```

Comment:
This is the only part I'm unsure about, I'm not sure if the `minRiskLimit` and `maxRiskLimit` are actually the `notionalFloor` and `notionalCap`